word_break.py

```diff
@@ -0,0 +1,55 @@
+'''
+______________Asumptions__________________
+
+1. Function returns false if subset of woard is not in DIctonary
+2. Words are seprated by blank space
+3. Funcrion returns false if dictonary is empty
+4. Function retuens False is words are not in Dictonary  
+5. Function returns false if input string is empty
+
+'''
+
+
+
+
+def word_break(dictonary,sentence,demiliter):
+	splitedWord=""
+	word=""
+	spacecount=0
+	for letter in sentence:
```

Comment:
This is a good start. One test case for you to consider is being given the string "cars" with the dictionary { "car", "cars" }